docs/tests-ui.rst

```diff
@@ -1,177 +1,151 @@
-Client-side Testing with Intern
-===============================
-From 2014 to 2016, we used Intern_ for client-side testing. It uses Selenium
-WebDriver API which lets us write automated testing via JavaScript. Intern is
-an open source project created and maintained by SitePen_.
+===================
+Client-side Testing
+===================
 
-In 2016, we started to convert our client-side tests to Python, to standardize
-on common Mozilla QA processes, and as part of an effort to add client-side
-tests to the development pipline.  The Intern tests described here are no
-longer run for development and deployment, and have not been updated for the
-Docker development environment.
+Kuma has a suite of functional tests using `Selenium`_ and `pytest`_. This allows us
+to emulate users interacting with a real browser. All these test suites live in
+the /tests/ directory.
 
-.. _SitePen: http://sitepen.com
-.. _Intern: https://theintern.github.io/
+The tests directory comprises of:
 
-Installing Dependencies
------------------------
+   * ``/functional`` contains pytest tests.
```

Comment:
Please remove the leading whitespace, or else it will be styled like a block quote.

<img width="427" alt="screenshot 2016-11-22 12 15 46" src="https://cloud.githubusercontent.com/assets/286017/20536115/6fa0d59c-b0ad-11e6-829a-0dceae40c613.png">